guides/running_a_campaign.md

```diff
@@ -2,24 +2,49 @@
 
 ## Assumptions
 
-- There is a topic for the campaign, usually someone has approached the Rust content team (or the Community team) directly.
-- Pick a [focus topic][link_focus_topics] and write a summary of what you would like the campaign to cover.
+- There is a topic for the campaign, usually someone has approached the [Rust content team][link_content_team] (or the [Community team][link_community_team]) directly. If not, you can pick a [focus topic][link_focus_topics] and write a summary of what you would like the campaign to cover.
 
-## Time line
+## Timeline
 
-- Initiation
-  - Write an introductory blog post on the relevant blog sites i.e. community.rs, blog.rust-lang.org, etc
-  - Raise a tracking [issue][link_tracking_issue] for the campaign.
-- Review cycle
+The days/weeks next to main bullet points are duration for these phases.
```

Comment:
I would add that this is a suggested timeline and this should be also documented in the campaigns readme.